README.md

```diff
@@ -49,6 +49,7 @@ To install the connector you will need to provide the following configuration se
 - `DOVETECH_API_HOST` - this is available on the Details tab of the Dovetech Customer Portal
 - `DOVETECH_API_KEY` - this is the Processor API Key from your project in the Dovetech Customer Portal
 - The commercetools settings prefixed with `CTP_` should be populated from the API client created above
+- `BASIC_AUTH_PASSWORD` - these two variables (current/previous) will allow the connector to validate that calls come from commercetools. `BASIC_AUTH_PASSWORD_CURRENT` is required, and has a minimum length of two characters and a maximum of 50. The contents will will be hashed for evaluation.
```

Comment:
Maybe remove the comment about hashing? It's not strictly hashed as I understand it - it's base64 encoded and then creates a buffer?

```suggestion
- `BASIC_AUTH_PASSWORD` - these two variables (current/previous) will allow the connector to validate that calls come from commercetools. `BASIC_AUTH_PASSWORD_CURRENT` is required, and has a minimum length of two characters and a maximum of 50.
```

Reply:
removed